sdcm/kcl_thread.py

```diff
@@ -45,7 +45,7 @@ def build_stress_cmd(self):
         return stress_cmd
 
     def _run_stress(self, loader, loader_idx, cpu_idx):
-        docker = RemoteDocker(loader, "scylladb/hydra-loaders:kcl-jdk8-20210215",
+        docker = RemoteDocker(loader, "scylladb/hydra-loaders:kcl-jdk8-20210526-ShardSyncStrategyType-PERIODIC",
```

Comment:
@yarongilor 
this is a mistake (the docker name contains a strange name)?

Reply:
well, we had to set a name to reflect what's new about it, so we just added it according what Israel already started.